### PR TITLE
Imported file move

### PR DESCRIPTION
**Pull request by cjh1**
If an imported file is moved to another assetstore we need to reset the imported flag. Otherwise, in the case of the filesystem assetstore the download will fail as the appropriate path will not be generated.

### CODE REVIEW

**Comment by cjh1** [comment]
@manthey PTAL

File: girder/models/upload.py [hunk 1]
```diff
@@ -178,6 +178,10 @@ def finalizeUpload(self, upload, assetstore=None):
             file['created'] = datetime.datetime.utcnow()
             file['assetstoreId'] = assetstore['_id']
             file['size'] = upload['size']
+            # If the file was previously import, it is no longer.
```
Comment (manthey) [inline girder/models/upload.py]: import -> imported

**Comment by cjh1** [comment]
@manthey Thanks for the review

**Comment by cjh1** [comment]
@TristanWright This should resolve the issue we were seeing yesterday.

**Pull request merged**